controllers/serviceaccount/serviceaccount_remover_test.go

```diff
@@ -0,0 +1,217 @@
+package serviceaccount_test
+
+import (
+	"context"
+	"github.com/cloudogu/cesapp/v4/core"
+	cesmocks "github.com/cloudogu/cesapp/v4/registry/mocks"
+	"github.com/cloudogu/k8s-dogu-operator/controllers/serviceaccount"
+	"github.com/cloudogu/k8s-dogu-operator/controllers/serviceaccount/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestRemover_RemoveServiceAccounts(t *testing.T) {
+	var postgresRemoveCmd core.ExposedCommand
+	for _, command := range postgresqlDescriptor.ExposedCommands {
+		if command.Name == "service-account-remove" {
+			postgresRemoveCmd = command
+			break
+		}
+	}
+	require.NotNil(t, postgresRemoveCmd)
+
+	t.Run("success", func(t *testing.T) {
+		// given
+		ctx := context.TODO()
+		doguConfig := &cesmocks.ConfigurationContext{}
+		doguConfig.Mock.On("Exists", "sa-postgresql").Return(true, nil)
+		doguConfig.Mock.On("DeleteRecursive", "sa-postgresql").Return(nil)
+		doguRegistry := &cesmocks.DoguRegistry{}
+		doguRegistry.Mock.On("IsEnabled", "postgresql").Return(true, nil)
+		doguRegistry.Mock.On("Get", "postgresql").Return(postgresqlDescriptor, nil)
+		registry := &cesmocks.Registry{}
+		registry.Mock.On("DoguConfig", "redmine").Return(doguConfig)
+		registry.Mock.On("DoguRegistry").Return(doguRegistry)
+		commandExecutorMock := &mocks.CommandExecutor{}
+		commandExecutorMock.Mock.On("ExecCommand", mock.Anything, "postgresql", "test", &postgresRemoveCmd, []string{"redmine"}).Return(nil, nil)
+		serviceAccountCreator := serviceaccount.NewServiceAccountRemover(registry, commandExecutorMock)
+
+		// when
+		err := serviceAccountCreator.RemoveServiceAccounts(ctx, redmineCr, redmineDescriptor)
```

Comment:
Your tests are very readable. I like that a lot because I can focus on the business value of the code :muscle: